README.rst

```diff
@@ -78,6 +78,40 @@ The ``asv preview`` command will give the URL of the local web server (e.g.
 http://127.0.0.1:21331) - go to this address in your favorite browser to see
 the results.
 
+Running benchmarks with specific versions of dependencies
+---------------------------------------------------------
+
+If you would like to run the benchmarks against specific version(s) of an
+installed dependency or dependencies, add the ``matrix.req`` entry to 
+``asv.conf.json``, including the version(s) you wish to run benchmarks against:
+
+.. code-block:: diff
```

Comment:
I don't think `diff` produces the correct rendering.
```suggestion
.. code-block:: json
```

Reply:
Oh wait,  nvm, you are trying to show a diff ... But still it is all underlined in preview.